rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.63.0"
+channel = "1.69.0"
```

Comment:
I would prefer to keep the MSRV to 1.63, to allow building this project using Debian stable (https://packages.debian.org/stable/rust/rustc).

Reply:
Got it, done!